src/SeedAddress.spec.ts

```diff
@@ -1,10 +1,8 @@
 import SeedAddress from "./SeedAddress.svelte";
-import { render } from "@testing-library/svelte";
-import "@public/index.css";
 
 describe("SeedAddress", () => {
   it("shows the seed emoji and seed host", () => {
-    render(SeedAddress, {
+    cy.mount(SeedAddress, {
       props: {
         seed: {
```

Comment:
I'm not sure why the error hasn't shown up before, but it seems we should cast the `seed` property to the `Seed` class as:
```diff
diff --git a/src/SeedAddress.spec.ts b/src/SeedAddress.spec.ts
index 4e4b4ed..b651967 100644
--- a/src/SeedAddress.spec.ts
+++ b/src/SeedAddress.spec.ts
@@ -1,3 +1,4 @@
+import type { Seed } from "@app/base/seeds/Seed";
 import SeedAddress from "./SeedAddress.svelte";
 
 describe("SeedAddress", () => {
@@ -8,7 +9,7 @@ describe("SeedAddress", () => {
           id: "hydkkkf5ksbe5fuszdhpqhytu3q36gwagj874wxwpo5a8ti8coygh1",
           emoji: "🐱",
           host: "seed.cloudhead.io",
-        },
+        } as Seed,
         port: 8776,
       },
     });
@@ -25,7 +26,7 @@ describe("SeedAddress", () => {
           id: "hydkkkf5ksbe5fuszdhpqhytu3q36gwagj874wxwpo5a8ti8coygh1",
           emoji: "🐱",
           host: "seed.cloudhead.io",
-        },
+        } as Seed,
         port: 8776,
         full: true,
       },
```

Reply:
It's weird that it wasn't caught by our CI. Will look into it.

Reply:
0bbdccdbe019377314ee4e3278f602d2e9399b32